backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/CertificationValidityChecker.java

```diff
@@ -79,6 +80,12 @@ private void checkCertificationValidity() {
                 return;
             }
 
+            // ovirt-provider-ovn certificate doesn't exist if OVN service is disabled during setup
+            checkOptionalCertificate(EngineLocalConfig.getInstance().getPKIOvirtProviderOVNCert(),
+                    AuditLogType.OVIRT_PROVIDER_OVN_CERTIFICATE_HAS_EXPIRED,
```

Comment:
We wouldn't get a warning for it if the engine certificate expires first, which might be misleading. Is that intended?

Reply:
Maybe -- we needn't spam with warnings about the expiration if the Engine / OVN certificate expiration dates are +- aligned.  But if it is the case then it should be mentioned in the commit message.

Reply:
Right, that  a nonsense, which we have in the code from the beginning. If engine CA is going to expire, then it needs to be renewed first and only afterwards it makes sense to renew other certificates. Pushed additional patch ...